packages/govuk-frontend/src/govuk/init.mjs

```diff
@@ -67,9 +74,10 @@ function initAll(config) {
  * @param {T} Component - class of the component to create
  * @param {T["defaults"]} [config] - config for the component
  * @param {Element|Document} [$scope] - scope of the document to search within
+ * @param {errorCallback<T>} [errorCallback] - scope of the document to search within
  * @returns {Array<InstanceType<T>>} - array of instantiated components
  */
-function createAll(Component, config, $scope = document) {
+function createAll(Component, config, $scope = document, errorCallback) {
```

Comment:
**suggestion** As discussed during our call, let's try to limit the number of arguments to 3 and regroup the scope and error callback in a param that can be either:
- an `Element|Document` representing the scope
- a `Function` representing the error callback
- an `{scope: Element|Document, errorCallback: Function}` if both are needed.